MakerDAOs-Bridge-Invariant-Bot/src/externalApiCall.ts

```diff
@@ -0,0 +1,12 @@
+import { getAlerts } from "forta-agent";
+export const callFortaAPI = async (botId: string, chainId?: number) => {
+  try {
+    const alretResponse = await getAlerts({
+      botIds: [botId],
+      chainId: chainId,
+    });
```

Comment:
```suggestion
    const alretResponse = await getAlerts({
      botIds: [botId],
      chainId: chainId,
      blockNumberRange: {
        startBlockNumber: blockNumber - 20,
        endBlockNumber: blockNumber,
      },
      first: 1,
    });
```

`blockNumberRange` should be passed into this call, so that it should work for a specific block number when running `npm run block ...`, 
also `first: 1`  means that only the most recent alert returns. (its default value is 5 but we only look for the last emitted alert anyway)